### DISCUSSION TITLE
Select, using objects as the values

### DISCUSSION THREAD

**Discussion by anton-johansson** [chakra-ui/chakra-ui#2282] [Q&A] [answered]
Hi!

I was wondering if it was possible to use objects as values, instead of `string` or `number`? So whenever an item is selected, I'd be able to get the entire object. Or is that something I'd have to handle myself?

**Comment by chasinhues** [comment, marked as answer]
You would have to handle this yourself, probably by just implementing an `onChange` handler for the `Select`.
